src/charts/CytoViz/components/DetailsLog/DetailsLog.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import useStyles from './style';
+
+function prettyPercentage(val) {
+  return (val * 100).toFixed(1) + '%';
+}
+
+function hiddenElements(cytoElements) {
+  let hiddenElements = 0;
+  cytoElements.forEach((element) => {
+    if (element.data('hidden')) {
+      hiddenElements++;
+    }
+  });
+  return hiddenElements;
+}
+
+const DetailsLog = (props) => {
+  const { cytoAsState } = props;
+  const classes = useStyles();
+
+  const compounds = cytoAsState.nodes().parent().length;
+  const children = cytoAsState.nodes().nonorphans().length;
```

Comment:
If these variables count a number of elements, could we rename them to `compoundsCount` and `childrenCount` please ? :)